### DISCUSSION TITLE
第一次体验到什么叫小鹿乱撞

### DISCUSSION THREAD

**Discussion by fangmouhenfang** [share121/inter-knot#430] [其他]
从之前工作的城市辞职后下乡隐居，结果刚搬来这地没多久，和邻居们都还不熟，先找到意中人了。之前也喜欢过很多虚拟角色，游戏和动漫都有，也一时上头当过xxx的狗，但是这次面对的是一个活生生的人啊。。。虽然他有时候会被别人吐槽有点阴暗，死气沉沉，但好歹是真人啊。。。
第一次遇见他时，他默默游离在人群边缘；第二次遇见他时，他独自一人在海边看雨。他总给我一种不好接近的印象，凭借这点我能够很轻松在人群中找到他（听邻居说是和家里关系不大好，人也内向）。后来某天偶然看到他的一张照片，一时失神，直接举着手机被硬控几十秒。脑海中仅存的理智试图让我保持冷静（你只是图人家好看而已），但心脏一直在砰砰跳，一想到他就有加速的趋势。结果最后越来越难受，胸口发闷，总感觉喉咙有什么东西梗着，体温明显上升，摸着比平常烫。。。小鹿乱撞？感觉都快把心肌撞破了，之前厨虚拟角色都不带这样的。
自那天后我常常问自己要不要试图去接近他，但我至今母单一直社畜，别人跟我表白我只会回个哦，也分不清到底什么是一时上头的心动和真的想要长相厮守。本以为我这辈子会一直单下去，自己也有意终身不婚独处到老，但每次看见他总会被勾起心跳的余波（难受，该不会是心脏病吧，不知道能不能找他索赔，算了有点离谱），让我一遍遍问自己到底是动了真心还是单纯颜狗。如果他真的如别人所说那样孤僻，我又能否在他心中荒芜开垦出一边沃土？
从前的我自诩独身主义嘲笑恋爱脑，现在的我离开城市感觉降智了不少。
![1722917804570](https://github.com/user-attachments/assets/17a9191a-7f0a-40db-acbd-f4691e43db07)

**Comment by kivss** [comment]
找个机会A上去！
